src/process.ts

```diff
@@ -174,20 +174,36 @@ export class PowerShellProcess {
         return true;
     }
 
-    private waitForSessionFile(): Promise<utils.IEditorServicesSessionDetails> {
-        return new Promise((resolve, reject) => {
-            utils.waitForSessionFile(this.sessionFilePath, (sessionDetails, error) => {
-                utils.deleteSessionFile(this.sessionFilePath);
+    private sleep(ms: number) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+    }
+
+    private async waitForSessionFile(): Promise<utils.IEditorServicesSessionDetails> {
```

Comment:
Thank you!  We've been using async/await in our Electron/Angular app code base and vastly prefer it to dealing with Promises directly.   :-)